pageObjects/NotesPage.js

```diff
@@ -17,12 +18,19 @@ class NotesPage extends BasePage {
         this.newNoteTitleField.click()
         this.newNoteTitleField.sendKeys(title)
         element(by.buttonText('Save')).click()
-        browser.sleep(browser.params.customTimeout)
+        if (title !== '' && body !== '') {
+            browser.wait(EC.visibilityOf(this.noteFirst), browser.params.customTimeout,
+                'Success notification should be visible after note creation')
+        }
     }
 
     archiveNote() {
         browser.actions().mouseMove(this.noteFirst).perform()
         this.iconArchive.click()
+        // this.notificationSuccess.getAttribute('innerHTML')
+        //     .then((text) => console.log('===== ' + text))
+        browser.wait(EC.visibilityOf(this.notificationSuccess), browser.params.customTimeout,
```

Comment:
browser.wait(EC.presenceOf(this.notificationSuccess), browser.params.customTimeout,
            'Success notification should be visible after note archivation')